Notes and Outlines/markdown modules/05-install.md

```diff
@@ -0,0 +1,52 @@
+# Installing Jekyll Locally
+
+This lesson will assume you know how to use Git locally on the command line. If you don't know how to do this, we'd recommend going through our Introduction to Git tutorial first.
+
+
+## Jekyll on Your Computer
+
+There are many benefits to installing Jekyll on your computer and working on your projects locally. You can more quickly test what your changes look like on the web without having to merge your branch to do so. You can start new projects from scratch instead of having to use our template. You'll be able to configure Jekyll to work for a wider variety of project types. And you can use your favorite text editor to make the process of writing your content that much faster and more pleasant.
+
+However, Jekyll can be difficult to set up. And it isn't supported on Windows, only on Mac and Linux. We're assuming you have a Mac computer for this tutorial: if you're using Windows, you may be able to get Jeckyll to work. Here are some [instructions](http://jekyllrb.com/docs/windows/). 
+
+Getting Jekyll set up locally and using GitHub pages will require several steps. Unfortunately, some of the most boring and finicky steps must come first, but you'll only need to perform them once to get started.
+
+1. Install Git
+2. Install Jekyll
```

Comment:
FYI: Not sure if you cover this elsewhere but we suggest users install Jekyll by installing the [GitHub Pages gem](, which sets up the ideal development environment for Jekyll and GitHub Pages. We also recommend users install Bundler first. These steps help prevent troubleshooting problems later on.